superflore/generators/ebuild/ebuild.py

```diff
@@ -94,9 +94,8 @@ def get_ebuild_text(self, distributor, license_text):
         Generate the ebuild in text, given the distributor line
         and the license text.
 
-        @todo: make the year dynamic
         """
-        ret = "# Copyright 2017 " + distributor + "\n"
+        ret = "# Copyright " + strftime("%Y", gmtime()) + distributor + "\n"
```

Comment:
You're missing whitespace after the year.
```
- # Copyright 2017Open Source Robotics Foundation
+ # Copyright 2017 Open Source Robotics Foundation
```

Reply:
This was caught by the testing but it shows that next year it will be a problem for the testing. The test will need to be extended with similar logic. It's hard coded to 2017 here: https://github.com/ros-infrastructure/superflore/blob/master/tests/ebuild/simple_expected.ebuild#L1

Reply:
Thank you so much for this feedback, I will get on changing these problems ASAP! :)